frontend/src/components/ProjectPage/ProjectPage.js

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import axios from "axios";
+import ProjectList from "./ProjectList";
+import ProjectCreate from "./ProjectCreate";
+
+class ProjectPage extends React.Component {
+  state = {
+    projects: null,
+    create: false,
+  };
+
+  async componentDidMount() {
+    const projects = await this.GetProjects();
+    const tags = await this.GetTags();
+    this.setState({ projects: projects, tags: tags });
+  }
+
+  async GetProjects() {
+    const response = await axios.get(this.props.url + "/api/projects/", {
+      headers: {
+        Authorization: "Token " + localStorage.getItem("token"),
+      },
+    });
+    return response.data;
+  }
+
+  async GetTags() {
+    const response = await axios.get(this.props.url + "/api/tags/", {
+      headers: {
+        Authorization: "Token " + localStorage.getItem("token"),
+      },
+    });
+    return response.data;
+  }
+
+  UpdateProjects = async () => {
+    const projects = await this.GetProjects();
+    this.setState({ projects: projects });
+  };
+
+  onCreateButtonClick() {
+    if (this.state.create) {
+      this.setState({ create: false });
```

Comment:
You can just return here and avoid writing else, not needed just my personal preference to always return instead of writing `else`